sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/MultiMasterConflictResolutionTest.java

```diff
@@ -82,8 +82,7 @@ public void conflictResolutionPolicyCRUD() {
             // when (e.StatusCode == HttpStatusCode.BadRequest)
             CosmosException dce = Utils.as(e, CosmosException.class);
             if (dce != null && dce.getStatusCode() == 400) {
-                assertThat(dce.getMessage()).contains("Invalid path '\\\\\\\\\\\\/a\\\\\\\\\\\\/b' for last writer " +
-                    "wins conflict resolution");
+                assertThat(dce.getMessage()).contains("Invalid path '\\\\/a\\\\/b' for last writer wins conflict resolution");
```

Comment:
Is it a regression? Why isn't it caught before? Where is the source for it? 

Isn't it fragile.